app.json

```diff
@@ -38,7 +38,7 @@
       },
       "NODE_TLS_REJECT_UNAUTHORIZED": {
         "description": "Because Heroku SSL is self signed, SSL fails if this is not set to 0",
-        "value": 0
+        "value": "0"
```

Comment:
Could you please explain why you propose to wrap it into "?
It seems that 0 parsed correctly

Reply:
It showed up in Heroku's Config Vars as an empty value in my experience